config/routes/v1x2.rb

```diff
@@ -38,7 +38,9 @@
     post :undelete, :action => 'undestroy', :controller => 'portfolio_items'
   end
   resources :icons, :only => [:create, :destroy]
-  resources :order_processes, :only => [:create, :destroy, :index, :show, :update], :concerns => [:taggable]
+  resources :order_processes, :only => [:create, :destroy, :index, :show, :update], :concerns => [:taggable] do
+    post :pre, :action => 'pre', :controller => 'order_processes'
```

Comment:
This is an `update` action, should we use `patch` as regular?

Reply:
Sure, that's fine